src/get-default-printer/get-default-printer.spec.ts

```diff
@@ -62,3 +64,29 @@ it("throws when execAsync fails", () => {
   mockedExecAsync.mockRejectedValue("error");
   return expect(getDefaultPrinter()).rejects.toBe("error");
 });
+
+it("gets the default printer with custom and repeated properties", async () => {
+  const stdout = `
+
+  Name                        : Microsoft Print to PDF
+  Caption                     : Microsoft Print to PDF
+  DeviceID                    : Microsoft Print to PDF
+  PaperSizesSupported         : {1, 1, 1, 1...}
+  PortName                    : USB001
+  PrinterPaperNames           : {USER, 144mm x 100mm}
+  
+  `;
+
+  mockedExecAsync.mockResolvedValue({
+    stdout,
+    stderr: "",
+  });
+
+  const result: Printer | null = await getDefaultPrinter();
+
+  expect(result).toStrictEqual({
+    deviceId: "Microsoft Print to PDF",
+    name: "Microsoft Print to PDF",
+    paperSizes: ["USER", "144mm x 100mm"],
```

Comment:
i am sorry for bothering you with such small things, but could you replace `USER` with `A4` for example?

Reply:
no problem, changed